polls/forms.py

```diff
@@ -0,0 +1,71 @@
+from datetime import datetime
+from django import forms
+from polls.models import UserProfile
+from django.core.exceptions import ValidationError
+from django.utils.translation import ugettext_lazy as _
+
+
+class UserCreateForm(forms.Form):
+    BIRTH_YEAR_CHOICES = []
+    for x in range(1900, int(datetime.now().year) + 1):
```

Comment:
to można zapisać 
BIRTH_YEAR_CHOICES = [x for x range(1900, int(datetime.now().year) + 1)]

https://www.python.org/dev/peps/pep-0289/